src/officehours/settings.py

```diff
@@ -40,7 +40,7 @@ def str_to_bool(val):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = str_to_bool(os.getenv('DEBUG', 'off'))
 
-ALLOWED_HOSTS = csv_to_list(os.getenv('ALLOWED_HOSTS', None))
+ALLOWED_HOSTS = csv_to_list(os.getenv('ALLOWED_HOSTS', 'localhost,127.0.0.1'))
```

Comment:
You should not hard-code `localhost` here for testing on your computer. Instead, you should set the value in the `ALLOWED_HOSTS` environment variable.